test/native/lib/Test.re

```diff
@@ -3,13 +3,6 @@ open Utils;
 open Css_types;
 open Helpers;
 
-let {describe, describeSkip, describeOnly} =
```

Comment:
It produced errors so I just removed this. Not sure what it was used for but tests pass :D

Reply:
This part was used to "clear the cache" of acceptableNames before each unit test was run, since the ref doesn't get reset until after all the tests have run. Since we're now doing actual caching logic, it kind of makes sense that the tests are now passing!